### PR TITLE
test: Add user mail testing.

### PR DESCRIPTION
**Pull request by appleboy**
as title.

cc @lunny @ethantkoenig

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

File: models/user_mail_test.go [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+	emails := make([]*EmailAddress, 2)
+	emails[0] = &EmailAddress{
+		Email:       "user1234@example.com",
+		IsActivated: true,
+	}
+	emails[1] = &EmailAddress{
+		Email:       "user5678@example.com",
+		IsActivated: true,
+	}
+	assert.NoError(t, AddEmailAddresses(emails))
```
Comment (ethantkoenig) [inline models/user_mail_test.go:69]: Not to be nitpicky, but this function tests both `AddEmailAddress(..)` and `AddEmailAddresses(..)`. Could we split it into two tests: `TestAddEmailAddress` and `TestAddEmailAddresses`?

File: models/user_mail_test.go [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+	emails[0] = &EmailAddress{
+		UID:   int64(2),
+		ID:    int64(3),
+		Email: "user2@example.com",
+	}
+	emails[1] = &EmailAddress{
+		UID:   int64(2),
+		Email: "user21@example.com",
+	}
+	assert.NoError(t, DeleteEmailAddresses(emails))
```
Comment (ethantkoenig) [inline models/user_mail_test.go:113]: ^

**Review by ethantkoenig — CHANGES_REQUESTED** [review]

**Comment by ethantkoenig** [comment]
@appleboy A couple small changes, then it will Look-Good-To-Me

File: models/user_mail_test.go (hunk 2 above)
Comment (appleboy) [reply models/user_mail_test.go:113]: Done! @ethantkoenig

File: models/user_mail_test.go (hunk 1 above)
Comment (appleboy) [reply models/user_mail_test.go:69]: Done! @ethantkoenig

**Comment by appleboy** [comment]
@ethantkoenig Waiting build result.

**Comment by ethantkoenig** [comment]
LGTM

**Review by ethantkoenig — APPROVED** [review]

**Pull request merged**